proto/alliance/alliance/query.proto

```diff
@@ -249,7 +249,6 @@ message QueryAllianceUnbondingsByDelegatorResponse {
   option (gogoproto.goproto_getters) = false;
 
   repeated UnbondingDelegation unbondings = 1 [(gogoproto.nullable) = false];
-  cosmos.base.query.v1beta1.PageResponse pagination = 2;
```

Comment:
why we remove this?

Reply:
Because we don't have a way to paginate that specific request and the amount of items to be recovered from the API is number_of_alliances * 7 (maximum undelegations)